engine/HAL/STM32F0/luos_hal_config.h

```diff
@@ -7,6 +7,29 @@
  * @author Luos
  * @version 0.0.0
  ******************************************************************************/
+/*
+ *
+ * BOOT_APP MODE FLASH
+ *
+ *
+ *             SHARED_MEMORY_ADDRESS
+ *             0x0800C000
+ *               |
+ *               |   APP_START_ADDRESS                  APP_END_ADDRESS
+ *               |   |0x0800C800                          |
+ *               |   |                                    |
+ *               v   v                                    v
+ *   +-----------+---+------------------------------------+
+ *   |   48Kb    |2Kb|           206Kb                    |
```

Comment:
all the MCU in the different familly has differente memory size I realy like this drawing but I think you should remove the app size and adapt the adress if needed

Reply:
OK